environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - python>=3.6
+  - python=3
```

Comment:
what's the effect of this ?

Reply:
None, travis/pipeline replace this line, and for users it just installs any python3, which is 3.7 by default in mini conda